pom.xml

```diff
@@ -891,18 +894,19 @@
 					<extensions>true</extensions>
 					<executions>
 						<execution>
-							<id>bundle-manifest</id>
-							<phase>process-classes</phase>
+							<id>default-bundle</id>
+							<phase>package</phase>
 							<goals>
-								<goal>manifest</goal>
+								<goal>bundle</goal>
 							</goals>
 							<configuration>
 								<instructions>
-									<Export-Package>*</Export-Package>
-									<Import-Package>org.apache.wicket*,
-										org.junit.jupiter*</Import-Package>
-									<DynamicImport-Package>*</DynamicImport-Package>
+									<Require-Capability>osgi.ee;filter:="(&amp;(osgi.ee=JavaSE)(version=11.0))"</Require-Capability>
+									<Export-Package>${osgi.export.package}</Export-Package>
+									<Import-Package>${osgi.import.package}</Import-Package>
+									<DynamicImport-Package>${osgi.dynamicimport.package}</DynamicImport-Package>
 									<_nouses>true</_nouses>
+									<!-- _noee>true</_noee -->
```

Comment:
This could be removed, right ?

Reply:
> OsgiClashingPackagesTest.collectProjectPackages() is failing.

I'll look into that.

> <Require-Capability>osgi.ee;filter:="(&amp;(osgi.ee=JavaSE)(version=11.0))"</Require-Capability>
> This probably should be version 17.0 for master and 11 for wicket-9.x

Does Wicket master have _runtime_ require on JDK 17?

> <!-- _noee>true</_noee -->
> This could be removed, right ?

This flag would prevent the Require-Capability header from being generated.

Reply:
For Wicket, I think it is safe to remove the Require-Capability header altogether. (ie.. use the _noee = true flag.

Web app users are going to need to know full stack to align JDK + Karaf + Pax Web + Wicket at all times. This header only provides a pre-wiring check to fast fail.

Reply:
> Does Wicket master have _runtime_ require on JDK 17?

Yes!